content/en/blog/2019/performance-best-practices/index.md

```diff
@@ -2,6 +2,7 @@
 title: "Best Practices: Benchmarking Service Mesh Performance"
 description: "Tools and guidance for evaluating Istio's data plane performance."
 publishdate: 2019-07-09
+lastupdatedate: 2019-09-05
```

Comment:
@geeknoid Do you mean add this? I'm not sure which key-name I should use here.

Reply:
Nope, it's `last_update`.

See <https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#updating-front-matter> for all the front-matter fields supported.